script/github-actions/check-deployability.js

```diff
@@ -96,7 +96,7 @@ const isAncestor = (commitA, commitB) => {
  */
 const isAheadOfLastFullDeploy = async (commitSha, env) => {
   const lastFullDeployCommitSha = await getLastFullDeployCommit(env);
-  return isAncestor(commitSha, lastFullDeployCommitSha);
+  return isAncestor(lastFullDeployCommitSha, commitSha);
```

Comment:
This accidentally got flipped when I updated the function to use the argument instead of the `GITHUB_SHA` 🤦